tests/ui/pattern/usefulness/integer-ranges/gap_between_ranges.rs

```diff
@@ -0,0 +1,117 @@
+#![feature(exclusive_range_pattern)]
```

Comment:
The only test I don't see in this file is for the case where the patterns are in a different order than expected:

```rust
match 42 {
    10.. => {}
    1..9 => {}
    _ =>{}
}
```

Reply:
There's one line 27: `m!(0u8, 31..=40, 20..30);`

Reply:
Oh! I missed it, disregard then.